src/ucp/rndv/rndv.c

```diff
@@ -846,7 +851,7 @@ UCS_PROFILE_FUNC_VOID(ucp_rndv_recv_frag_put_completion, (self),
                 rreq->recv.remaining, freq->send.length);
     rreq->recv.remaining -= freq->send.length;
     if (rreq->recv.remaining == 0) {
-        ucp_rndv_recv_req_complete(rreq, UCS_OK);
+        ucp_request_complete_and_dereg_recv_rndv(rreq, UCS_OK);
```

Comment:
why needed to rename this function?

Reply:
since this is rndv-related function, IMO better move it to rndv.h and keep old name